src/shared/config.ts

```diff
@@ -24,60 +24,91 @@
  - Name Surname <name.surname@gatesfoundation.com>
 
  - Paweł Marzec <pawel.marzec@modusbox.com>
- - Abhimanyu Kapur <abhi.kapur09@gmail.com>
+ - Raman Mangla <ramanmangla@google.com>
```

Comment:
why you drop @akapur99 ?

Reply:
Not me who did this :( it was @ramanmangla

Reply:
You can see here: https://github.com/mojaloop/auth-service/pull/37/files
I will add him back :)